src/runtime_src/core/tools/xbmgmt2/SubCmdProgram.cpp

```diff
@@ -151,10 +161,20 @@ update_SC(unsigned int  index, const std::string& file)
   //versal flow to flash sc
   try {
     uint32_t val = xrt_core::query::program_sc::value_type(1);
+    bool done = false;
+    // Print progress while sc is flashed
+    std::thread t(progressReporter, std::ref(done));
     xrt_core::device_update<xrt_core::query::program_sc>(dev.get(), val);
+    done = true;
+    t.join();
+    std::cout << std::endl;
+	std::cout << boost::format("%-8s : %s \n\n") % "INFO" % "SC firmware image has been programmed successfully.";
```

Comment:
indentation fix. why don't you merge both std::couts?

Reply:
sure @chvamshi-xilinx